back-end-express/test_database/HotDog.sql

```diff
@@ -28,17 +28,19 @@ CREATE TABLE IF NOT EXISTS Customers(
   `LastName`   VARCHAR(45) NOT NULL,
   `Phone`      VARCHAR(45) NOT NULL,
   `Email`      VARCHAR(45) NOT NULL,
+  `Password`   VARCHAR(45) NOT NULL,
```

Comment:
don't we need username and password for admins and vendors?

Reply:
You are right, I will add those.